Source/Core/Core/HW/GCMemcard/GCMemcard.cpp

```diff
@@ -1568,11 +1568,9 @@ void InitializeHeaderData(HeaderData* data, const CardFlashId& flash_id, u16 siz
   }
   data->m_sram_bias = rtc_bias;
   data->m_sram_language = sram_language;
-  // TODO: determine the purpose of m_unknown_2
+  // TODO: determine the purpose of m_dtv_status
```

Comment:
Is this comment still necessary (or rather; something we could/should address here?) with the revalation that it appears to be the dtv status (guessing: component cable attached?)

Although it seems a bit odd that it would store that in the memory card; especially with the comment below that setting this flag only works for slot A.

Reply:
As I said I don't know if this old comment still apply. I'm almost certain that at least two of my GC games store the u16 value (promoted into u32) from 0xCC00206E (which happens to be VI DTV status register aka `_viReg[55]`). I can provide screenshots of the disassembled `__CARDFormatRegionAsync` function if needed.

The real question is rather how the testing was done, against which game/region. There is also a big chance that other region related settings might interfere with this one and is the cause of this misunderstanding.

Reply:
Then we should probably still keep that comment :)